### DISCUSSION TITLE
How do I default the system to Parallel 1P OD?

### DISCUSSION THREAD

**Discussion by V-FEXrt** [darthcloud/BlueRetro#741] [Q&A] [answered]
Hi! 

I'm flashing `BlueRetro_parallel_1p_3v3.bin` and I have to manually change the setting to Parallel_1P_OD every time I flash the ESP32.

The [documentation](https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-System-Specific-User-Manual#system-config) says 

```
If using universal firmware, it is required to set System to Parallel_1P_PP or Parallel_1P_OD in the global config and power cycle BlueRetro. Parallel 1P (12 buttons) mode can't be auto-detected.
```

which implies that the system is only required to be set when using the universal firmware but that shouldn't effect the parallel_1p specific firmware right?

Am I doing something wrong?/Is there a way to default to Parallel_1P_OD without going into the setting on a computer?

Thanks for taking the time to read!

**Comment by darthcloud** [comment]
This line if for "universal" firmware.

if you use specific firmware, dont touch the system option, leave it auto.

**Comment by V-FEXrt** [comment]
When flashing the specific firmware `v1.8.3_hw1/BlueRetro_parallel_1p_3v3.bin` the controller will only work if I manually change the setting. If I leave it as auto it doesn't work. Maybe this is a bug?

**Reply by darthcloud** [reply]
I will have to check, sound like a bug

**Reply by darthcloud** [reply]
can you grab the log from usb?
https://github.com/darthcloud/BlueRetro/wiki/Getting-BlueRetro-debug-logs-via-Serial-port-Windows-10

**Comment by V-FEXrt** [comment]
[before.txt](https://github.com/darthcloud/BlueRetro/files/12308427/before.txt)
[after.txt](https://github.com/darthcloud/BlueRetro/files/12308428/after.txt)

before.txt is the logs after a factory reset. The controller doesn't work

after.txt is the logs after changing from AUTO to Parallel_1P_OD at https://blueretro.io/advance.html and the controller does work

**Reply by darthcloud** [reply]
yeah its weird before.txt behave like universal firmware and is in detection mode and somehow detect PlayStation.

something wrong somewhere in that firmware defines.

**Comment by darthcloud** [comment, marked as answer]
It was a typo in the config for that specific build

That one should work:
[v1.8.3-138-g8831e52_hw1.zip](https://github.com/darthcloud/BlueRetro/files/12311789/v1.8.3-138-g8831e52_hw1.zip)

**Reply by V-FEXrt** [reply]
Awesome, thanks! I'll try it out after work tonight.

**Reply by V-FEXrt** [reply]
Yep! That fixed it! Thanks!
